quick_tour/the_view.rst

```diff
@@ -68,7 +68,7 @@ on its type:
 
     {# 1. Simple variables #}
     {# $this->render('template.html.twig', array(
-           'name' => 'Fabien')
+           'name' => 'Fabien'),
```

Comment:
This looks wrong ... maybe keep the new comma here ... but move the closing parenthesis to the next line:

```
{# $this->render('template.html.twig', array(
    'name' => 'Fabien',
)) #}
```